pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -492,6 +497,34 @@ func archiveHandler(mountPoint string) http.Handler {
 	})
 }
 
+func checkDirectoryPermissions(filePath string) bool {
+	dir, err := os.Open(filePath)
+	if err != nil {
+		log.Log.Reason(err).Errorf("error opening %s", filePath)
+		return false
+	}
+	defer dir.Close()
+
+	// Read all filenames
+	contents, err := dir.Readdirnames(-1)
+	if err != nil {
+		log.Log.Reason(err).Errorf("failed to read directory contents: %v", err)
+		return false
+	}
+
+	for _, item := range contents {
```

Comment:
I guess we cannot have subdirectories here?

Reply:
Right, with the current implementation we won't have any subdirectories here. This was discussed with the team and we decided there was no need to check permissions recursively.